android/expoview/src/main/java/host/exp/exponent/notifications/channels/ChannelScopeManager.java

```diff
@@ -0,0 +1,55 @@
+package host.exp.exponent.notifications.channels;
+
+import android.content.Context;
+
+import java.util.concurrent.Future;
+
+public class ChannelScopeManager implements ChannelManager {
```

Comment:
This looks like unused class. Is it safe to remove? Removing unused classes prior to review makes reviewing easier!

Reply:
It will be used in the Expo client. It's the first class in the chain pattern. ChannelScopeManager will scope channels.

Reply:
Let's add integration with Expo Client to this PR already — it will be easier to see what is used where